hieradata_aws/class/jenkins.yaml

```diff
@@ -196,7 +196,7 @@ govuk_jenkins::plugins:
   simple-theme-plugin:
     version: '0.5.1'
   slack:
-    version: '2.20'
+    version: '2.3'
```

Comment:
Is this a downgrade or an upgrade?

Reply:
My mistake, I removed that change, thanks!